pymobiledevice3/cli/developer.py

```diff
@@ -155,6 +155,14 @@ def kill(service_provider: LockdownClient, pid):
         ProcessControl(dvt).kill(pid)
 
 
+@dvt.command('processIdForBundleId', cls=Command)
```

Comment:
```suggestion
@dvt.command(cls=Command)
```

Reply:
This is non-standard for unix command to be in camelCase and the current convention is to be as described in the suggestion